packages/gatsby-source-wordpress/gatsby-node.js

```diff
@@ -0,0 +1,700 @@
+'use strict';
```

Comment:
This shouldn't be committed — related to the needed changes in .gitignore/.npmignore